src/pages/MainPage.tsx

```diff
@@ -0,0 +1,168 @@
+import useSWR from "swr";
+import { pagination, totalPageNumber } from "../functions/pagination";
+import { fetcher } from "../functions/fetcher";
+import { Data } from "../types/type";
+import { useSearchParams } from "react-router-dom";
+import { useEffect, useState } from "react";
+import { ChevronUpIcon, ChevronDownIcon } from "@heroicons/react/24/solid";
+
+const MainPage = () => {
+  const { data, error } = useSWR("/", fetcher);
+  const [state, setState] = useState<Data[]>([]);
+  const [isSortById, setIsSortById] = useState(true);
+  const [isSortByStatus, setIsSortByStatus] = useState(false);
+  const [isSortByDate, setIsSortByDate] = useState(true);
+  const [searchParams, setSearchParams] = useSearchParams({ page: "1" });
+
+  useEffect(() => {
+    if (data) {
+      setState(data);
+    }
+  }, [data]);
+
+  if (error) {
+    return <div>Something wrong...</div>;
+  }
+  return (
+    <section>
+      <article className="flex flex-col items-center">
+        <h1 className="text-4xl font-bold my-6">스위치원 주문 목록표</h1>
+        <table className="table w-3/4">
+          <thead>
+            <tr>
+              <th
+                className="flex"
+                onClick={() => {
+                  setIsSortByDate(true);
+                  setIsSortByStatus(false);
+                  setIsSortById(!isSortById);
+                  if (isSortById) {
+                    setState(
+                      state?.sort((a, b) => {
+                        return b.id - a.id;
+                      })
+                    );
+                  } else {
+                    setState(
+                      state?.sort((a, b) => {
+                        return a.id - b.id;
+                      })
+                    );
+                  }
+                }}
```

Comment:
```suggestion
  setState((state) => state.sort((a,b) => isSortById ? b.id - a.id : a.id - b.id))
```
이렇게 쓰면 코드가 좀 더 짧아질거같은데 
읽기가 불편할까요??